dozer/cogs/levels.py

```diff
@@ -0,0 +1,584 @@
+            if enabled:
+                embed.add_field(name="Success!", value=f"Server Levels Configured to these settings\n"
+                                                       f"XP min: {xp_min}\n"
+                                                       f"XP max: {xp_max}\n"
+                                                       f"Cooldown: {xp_cooldown} Seconds\n"
+                                                       f"Notification channel: {lvl_up_msgs}")
+            else:
+                embed.add_field(name="Success!", value="Server Levels Disabled")
+            await ctx.send(embed=embed)
+
+    configureranks.example_usage = """
+        `{prefix}configureranks --min 5 --max 15 --cooldown 15:` To configure rankings with a min value of 5, max value of 15 and a cooldown of 15 seconds
+        `{prefix}configureranks --disable:` To disable rankings
+        """
+
+    @command()
+    @guild_only()
+    async def getlevelsconfig(self, ctx):
+        """Returns the level settings for the current guild"""
+        settings = self._guild_settings[ctx.guild.id]
+
+        embed = discord.Embed(color=blurple)
+        embed.set_footer(text='Triggered by ' + ctx.author.display_name)
+
+        notify_channel = ctx.guild.get_channel(settings.lvl_up_msgs)
+
+        enabled = "Enabled" if settings.enabled else "Disabled"
+        embed.set_author(name=ctx.guild, icon_url=ctx.guild.icon_url)
+        embed.add_field(name=f"Levels are {enabled} for {ctx.guild}", value=f"XP min: {settings.xp_min}\n"
+                                                                            f"XP max: {settings.xp_max}\n"
+                                                                            f"Cooldown: {settings.xp_cooldown} Seconds\n"
+                                                                            f"Notification channel: {notify_channel}")
+
+        await ctx.send(embed=embed)
+
+    getlevelsconfig.example_usage = """
+        `{prefix}getlevelsconfig:` Returns levels config
+    """
+
+    @command()
+    @guild_only()
+    @discord.ext.commands.cooldown(rate=1, per=10, type=discord.ext.commands.BucketType.user)
+    async def rank(self, ctx, *, member: discord.Member = None):
+        """Get a user's ranking on the XP leaderboard.
+        If no member is passed, the caller's ranking is shown.
+        """
+        member = member or ctx.author
+
+        cache_record = await self._load_member(ctx.guild.id, member.id)
+
+        # Make Postgres compute the rank for us (need WITH-query so rank() sees records for every user)
+        db_record = await db.Pool.fetchrow(f"""
+            WITH ranked_xp AS (
+                SELECT user_id, rank() OVER (ORDER BY total_xp DESC) FROM {MemberXP.__tablename__}
+                WHERE guild_id = $1
+            ) SELECT rank FROM ranked_xp WHERE user_id = $2;
+        """, ctx.guild.id, member.id)
+
+        total_xp = cache_record.total_xp
+
+        count = ctx.guild.member_count
```

Comment:
It would be more accurate to fetch this from the database as number of records with this guild ID. As-is, it's possible to show e.g. "# 100 of 90" if enough high-ranked users have left the server.

Reply:
I added a mix of both uses the server size when the database only doesn't have the full server, and a the database total when the database is bigger than total members